### DISCUSSION TITLE
Get Started with Wio Terminal | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#203] [Q&A]
# Get Started with Wio Terminal | Seeed Studio Wiki

The Wio Terminal is a SAMD51-based microcontroller with Wireless Connectivity powered by Realtek RTL8720DN that’s compatible with Arduino and MicroPython. Currently, wireless connectivity is only supported by Arduino. It runs at 120MHz (Boost up to 200MHz), 4MB External Flash and 192KB RAM. It supports both Bluetooth and Wi-Fi providing backbone for IoT projects. The Wio Terminal itself is equipped with a 2.4” LCD Screen, onboard IMU(LIS3DHTR), Microphone, Buzzer, microSD card slot, Light sensor, and Infrared Emitter(IR 940nm). On top of that, it also has two multifunctional Grove ports for Grove Ecosystem and 40 Raspberry pi compatible pin GPIO for more add-ons.

https://wiki.seeedstudio.com/Wio_LTE_Cat.1

**Comment by BrewingGeek** [comment]
So to understand your statement above, you have a library which you use the same name as another library but want the user to go rename libraries because of the choice to use the same name as a previous library..    

 FYI   the example produces ERRORs and doe NOT work

**Reply by MatthewJeffson** [reply]
Hi! Thank you very much for your reminder. In fact, many of our wikis have not been updated for a long time. I will write this down and record it in our plans.
If you have product problems you can [email](https://www.seeedstudio.com/contacts) us, and you can also show us an image of what errors you have. Me and my colleague might help you with it.

**Comment by Brandtz1** [comment]
Hi again, anyone could help with this? 

(major part of error omitted) 

sketcheddesktop\calibrate\calibrate.ino:4:10: fatal error: Seeed_Arduino_LCD.h: No such file or directory
 #include <Seeed_Arduino_LCD.h>
          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.

Compilation error: Seeed_Arduino_LCD.h: No such file or directory

**Comment by gabogt4** [comment]
Hi, Can I power the wio terminal in header interface on pin 2 with 5 VDC from generic power supply?
